helpers/checkOfDraftOrFuturePost.ts

```diff
@@ -0,0 +1,19 @@
+import { PostDocumentWithoutContent } from "interfaces";
+
+const today = new Date();
+
+export function isPostADraft(post: PostDocumentWithoutContent) {
+  if (post.draft === true) {
+    return true;
+  } else {
+    return false;
+  }
+}
+
+export function isPostInTheFuture(post: PostDocumentWithoutContent) {
+  if (today.toISOString() < post.date) {
+    return true;
+  } else {
+    return false;
+  }
+}
```

Comment:
Still not quite right:

![image](https://user-images.githubusercontent.com/86713661/201299242-0e4f1506-6bc2-42ef-9acd-682b4932a44f.png)

Maybe because toISOString converts type, and it fucks the comparison